production-v2.cfg

```
@@ -13,12 +13,17 @@
 # It is not a replacement for the production.cfg but an extension.
 
 [buildout]
+early-parts += fix-solr-permissions
 parts += setpermissions
 
-
 [deployment]
 etc-directory = /apps/etc
 
+[fix-solr-permissions]
+recipe = plone.recipe.command
+command =
+    sudo -u ${buildout:os-user} chmod -R g+w ${buildout:directory}/parts/solr-instance/solr-webapp/webapp
```

```suggestion
    sudo -u ${buildout:os-user} chmod -R --silent g+w ${buildout:directory}/parts/solr-instance/solr-webapp/webapp
```
If no solr is installed, there is no such directory. I suggest to use `--silent` in order to indicate that there is no problem when the directory is missing; we want a `0` exit code anyway.

---

Good point. I saw that but thought it might be a good thing to output anyway. But actually we do not need to show this...